src/test/java/com/flightstats/hub/channel/ChannelValidatorTest.java

```diff
@@ -10,29 +10,34 @@
 import org.joda.time.DateTime;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
+import static com.flightstats.hub.model.ChannelType.BATCH;
+import static com.flightstats.hub.model.ChannelType.BOTH;
+import static com.flightstats.hub.model.ChannelType.SINGLE;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.fail;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
 
 class ChannelValidatorTest {
 
-    private ChannelService channelService;
+    @Mock
+    private Dao<ChannelConfig> channelConfigDao;
     private ChannelValidator validator;
 
     @BeforeEach
     void setUp() {
-        channelService = mock(ChannelService.class);
-        validator = new ChannelValidator(channelService);
-        when(channelService.channelExists(any(String.class))).thenReturn(false);
+        initMocks(this);
+        validator = new ChannelValidator(channelConfigDao);
```

Comment:
I'd change this to @ExtendWith the MockitoExtension.

Reply:
Yes I agree with you. ClusterContentService and ChannelService requires some serious refactoring. I realized it was too much work in the current ticket and out of scope for this ticket. We should definitely address it in another ticket.